api/server/controllers/assistants/v1.js

```
@@ -18,18 +20,37 @@ const createAssistant = async (req, res) => {
     const { openai } = await getOpenAIClient({ req, res });
 
-    const { tools = [], endpoint, conversation_starters, ...assistantData } = req.body;
+    const {
+      tools = [],
+      endpoint,
+      conversation_starters,
+      append_current_datetime,
+      ...assistantData
+    } = req.body;
     delete assistantData.conversation_starters;
+    delete assistantData.append_current_datetime;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)